lib/terraforming/cli.rb

```diff
@@ -242,7 +247,13 @@ def configure_aws(options)
 
     def execute(klass, options)
       configure_aws(options)
-      result = options[:tfstate] ? tfstate(klass, options[:merge]) : tf(klass)
+
+      subcommand_options = options.select { |k, v| OPTIONS_AVAILABLE_TO_SUBCOMMANDS.include? k }
```

Comment:
Do you mind if I blindly passed all the `options` as `subcommand_options` here?
In that case we can simplify the implementation by removing `OPTIONS_AVAILABLE_TO_SUBCOMMANDS` and its relevant code.